### PR TITLE
Optimised GPKG working copies for large datasets.

### PR DESCRIPTION
**Pull request by olsen232**
## Description

Sets page_size to 64KB - this is on the GPKG itself.
Sets cache_size to 200MB - this is done each time a session is opened.

### CODE REVIEW

**Comment by olsen232** [comment]
cc @hamishcampbell

File: sno/sqlalchemy/create_engine.py [hunk 1]
```diff
@@ -16,6 +16,8 @@
 from sno.geometry import Geometry
 from sno.exceptions import NotFound, NO_DRIVER
 
+GPKG_CACHE_SIZE_MB = 200
```
Comment (craigds) [inline sno/sqlalchemy/create_engine.py:19]: make it in MiB, then the maths below makes sense (the pragma value is in KiB)

**Review by craigds — COMMENTED** [review]
where'd the numbers you chose come from?

File: sno/sqlalchemy/create_engine.py (hunk 1 above)
Comment (olsen232) [reply sno/sqlalchemy/create_engine.py:19]: Done

**Comment by rcoup** [comment]
Benchmark tests show any difference?

**Comment by hamishcampbell** [comment]
Huge improvement.

3M+ dataset before the change:

```
$ time sqlite3 data.gpkg "SELECT COUNT(building_id) FROM building_outlines;"
3320006
sqlite3 data.gpkg   0.94s user 5.60s system 10% cpu 1:04.05 total
```

After the change:

```
$ time sqlite3 data.gpkg "SELECT COUNT(building_id) FROM building_outlines;"
3320006
sqlite3 data.gpkg   0.24s user 0.25s system 99% cpu 0.488 total
```

**Comment by olsen232** [comment]
Glad it's working! I found and fixed our benchmarks for importing and writing GPKGs, but nothing in particular showed up amongst the noise. Most of our benchmarks measure sno internal performance, not GPKG performance.

**Pull request merged**